my-app/src/app/components/load-more-button/load-more-button.component.scss

```diff
@@ -0,0 +1,7 @@
+.load-more-button {
```

Comment:
Do you need this style? Maybe you can just create a rule `button {...}`

Reply:
Fixed and most likely it will be great to create common button component in one of the next MRs to reuse in the different place